connectors/src/connectors/slack/bot.ts

```diff
@@ -189,13 +190,59 @@ async function botAnswerMessage(
       }
     }
   }
+  // Extract all ~mentions.
+  const mentionCandidates = message.match(/~(\S+)/g) || [];
```

Comment:
"summarize ~gpt!" maybe we want candidates to be ["gpt"] instead of ["gpt!"] ? Not strongly held just thinking out loud

Reply:
Ok thanks. I looked at the regexp to validate an assistant name and I am now extracting accordingly.